### PR TITLE
[doc] Updated github button & removed announcement

### PR DESCRIPTION
**Pull request by shijianjian**

### CODE REVIEW

**Comment by shijianjian** [comment]
Probably leave the responsive GitHub button colour changing into next PRs. Need to get the hook from Furo. Not going to do it now.

**Comment by shijianjian** [comment]
worked locally. can be merged

File: docs/source/conf.py [hunk 1]
```diff
@@ -110,19 +110,22 @@
...
     "navigation_with_keys": True,
     'display_version': False,
-    "announcement": """
-        <a style=\"text-decoration: none; color: white;\"
-           href=\"https://github.com/kornia/kornia\">
-           <img src=\"https://github.com/kornia/data/raw/main/GitHub-Mark-Light-32px.png\" width=20 height=20/>
-           Star Kornia on GitHub
-        </a>
-    """,
+    # "announcement": """
```
Comment (edgarriba) [inline docs/source/conf.py:122]: Maybe we could link to our YouTube channels?

**Comment by edgarriba** [comment]
@shijianjian please merge

**Review by edgarriba — APPROVED** [review]

**Pull request merged**